### PR TITLE
fix grammar

### PR DESCRIPTION
**Pull request by AnilSeervi**
fix grammar

### CODE REVIEW

File: doc/spec.md [hunk 1]
```diff
@@ -255,7 +255,7 @@ The TypeScript type system enables programmers to express limits on the capabili
 var i = 0;
 ```
 
-TypeScript will infer from the following function definition that the function f has return type string.
+TypeScript will infer from the following function definition that the function 'f' has return type string.
```
Comment (AnilSeervi) [inline doc/spec.md:258]: The function name is referred in single quotes in the whole doc.

File: doc/spec.md [hunk 2]
```diff
@@ -255,7 +255,7 @@ The TypeScript type system enables programmers to express limits on the capabili
 var i = 0;
 ```
 
-TypeScript will infer from the following function definition that the function f has return type string.
```
Comment (AnilSeervi) [inline doc/spec.md:258]: The function name is referred in single quotes in the whole doc.

File: doc/spec.md [hunk 3]
```diff
@@ -965,7 +965,7 @@ The ***root container*** of an entity is defined as follows:
 
 Intuitively, the root container of an entity is the outermost module or namespace body from within which the entity is reachable.
 
-Interfaces, enums, and namespaces are "open ended," meaning that interface, enum, and namespace declarations with the same qualified name relative to a common root are automatically merged. For further details, see sections [7.2](#7.2), [9.3](#9.3), and [10.5](#10.5).
```
Comment (AnilSeervi) [inline doc/spec.md:968]: open ended is supposed to be written as 'open-ended'.

**Review by sandersn — CHANGES_REQUESTED** [review]
The spec is no longer maintained, so we're not accepting PRs to it.

**Pull request closed without merging**